### ISSUE TITLE
Cloud Functions

### ISSUE THREAD

**Issue by dgurney75** [distriqt/ANE-Firebase#218]
Can you look into adding cloud functions to the Firebase ANE?

**Comment by marchbold** [comment]
Have you got some references to the native functionality you want added?

**Comment by dgurney75** [comment]
Right now I just need to be able to connect to the secure REST endpoints used by the Firebase cloud functions. Can’t seem to find a way to connect using the httpClient libraries I have. I can send you  a sample endpoint to try if you want to try it yourself.




Dan Gurney
(M) 314-566-2325
dgurney75@gmail.com

________________________________

**Comment by marchbold** [comment]
Yeah any references you have to the documentation of the functionality you are looking for would be helpful for us

**Comment by dgurney75** [comment]
Take a look at what MyFlashLabs has for cloud functions. That's basically what is needed. Would be nice to add other methods of calling them besides onCall();

**Comment by marchbold** [comment]
No problems, we have this in development and are hoping a release soon. If you have any specific references to functionality you require please let us know.

**Comment by marchbold** [comment]
Reference: https://firebase.google.com/docs/functions
